packages/site/src/layout/LeftDrawer.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Button } from "@jobber/components";
+import { PropsWithChildren } from "react";
+import styles from "./LeftDrawer.module.css";
+
+interface LeftDrawerProps extends PropsWithChildren {
+  readonly open: boolean;
+  readonly onClose: () => void;
+  readonly header?: React.ReactNode;
+}
+
+export function LeftDrawer({
+  children,
+  open,
+  onClose,
+  header,
+}: LeftDrawerProps) {
+  if (!open) return null;
+
+  return (
+    <>
+      <div className={styles.overlay} onClick={onClose} />
+      <div className={styles.drawer}>
+        <Box padding="small" direction="row" alignItems="center" gap="small">
```

Comment:
if we make this `padding="base"` the position of the Jobber logo more or less stays in the same place

there's still a 2px difference between the two for some reason.... overall though, it looks nicer.

![Kapture 2025-02-21 at 14 44 57](https://github.com/user-attachments/assets/7d67417a-a099-4b7f-ad5e-deadff4c753f)

vs

![Kapture 2025-02-21 at 14 45 46](https://github.com/user-attachments/assets/22b16d92-8ee5-47d1-bc46-a4546bf0fd4d)